src/doc/trpl/match.md

```diff
@@ -23,7 +23,7 @@ match x {
 `match` takes an expression and then branches based on its value. Each *arm* of
 the branch is of the form `val => expression`. When the value matches, that arm's
 expression will be evaluated. It's called `match` because of the term 'pattern
-matching', which `match` is an implementation of.
+matching,' which `match` is an implementation of.
```

Comment:
no worries. i'm working on the comment doc style RFC, it just hasn't been merged, so you can't know. :/